src/components/Tasks.tsx

```diff
@@ -0,0 +1,38 @@
+import { Card } from 'antd'
+// import * as R from 'ramda'
+import React, { FunctionComponent } from 'react'
+// import { listData } from '../content'
+import { Draggable } from 'react-beautiful-dnd'
+
+export const Task: FunctionComponent<{ title: string; text: string }> = ({
+  title,
+  text,
+}) => (
+  <Card title={title} style={{ width: 300 }}>
+    <p>{text}</p>
+  </Card>
+)
+
+// const mapEntriesToTask = R.map(({ title, text }) => (
+//   <Task key={title} title={title} text={text} />
+// ))
+
+// export const Tasks: any = R.pipe<any, any, any, any>(
+//   R.path(['column', 'entries']),
+//   mapEntriesToTask,
+//   R.tap(console.log),
+// )(listData)
+
+export const Tasks = () => (
+  <div>
+    <Card title={'First'} style={{ width: 300 }}>
```

Comment:
```suggestion
    <Card title="First" style={{ width: 300 }}>
```
It is also correct syntax but easier to read.